### PR TITLE
homepage layout

### PR DESCRIPTION
**Pull request by Chiragmodi01**
#### Homepage layout
- Functional drawer
- Navbar with search
- Chips sections
- Video listings
image

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,7 +1,20 @@
+import './App.css';
+
+import Navbar from './comps/Navbar/Navbar'
+import Navpills from './comps/Navpills/Navpills'
+import Sidebar from './comps/Sidebar/Sidebar'
+import FooterNav from './comps/FooterNav/FoooterNav'
+import Home from './pages/Home/Home';
+
```
Comment (agrawalshanu708) [inline src/App.js:8]: use index.js and import from single file, thats make code clean

**Review by agrawalshanu708 — APPROVED** [review]
well structure code
you can merge it

File: src/comps/TempCard/TempCard.css [hunk 2]
```diff
@@ -0,0 +1,49 @@
+.tempCard-container {
+    flex-direction: column;
+    gap: .7rem;
+    width: 15.85rem;
+}
+.tempCard-header {
+    height: 8.9rem;
+    width: 100%;
+    background-color: #2c2c2c;
```
Comment (aurobindoGupta) [inline src/comps/TempCard/TempCard.css:9]: variable

**Review by aurobindoGupta — APPROVED** [review]
nice work.
clean code.

**Pull request merged**